test/ini_files/cgap_alpha.ini

```diff
@@ -1,11 +1,14 @@
 [app:app]
 use = config:base.ini#app
 session.secret = %(here)s/session-secret.b64
+auth0.client = ${AUTH0_CLIENT}
+auth0.secret = ${AUTH0_SECRET}
 file_upload_bucket = ${S3_BUCKET_ORG}-${S3_BUCKET_ENV}-files
 file_wfout_bucket = ${S3_BUCKET_ORG}-${S3_BUCKET_ENV}-wfoutput
 blob_bucket = ${S3_BUCKET_ORG}-${S3_BUCKET_ENV}-blobs
 system_bucket = ${S3_BUCKET_ORG}-${S3_BUCKET_ENV}-system
 metadata_bundles_bucket = ${S3_BUCKET_ORG}-${S3_BUCKET_ENV}-metadata-bundles
+sentry_dsn = ${SENTRY_DSN}
```

Comment:
And this must be replicated in cgap-portal?

Reply:
@willronchetti Yes, This is how I'd imagine it would be done in cgap-portal.

Reply:
Of course, something would then have to know to use it from there.